src/parser/parseReactElement.js

```diff
@@ -13,6 +13,7 @@ import type { TreeNode } from './../tree';
 const supportFragment = Boolean(Fragment);
 
 const getReactElementDisplayName = (element: ReactElement<*>): string =>
+  (element.type.render && element.type.render.name) ||
```

Comment:
![image](https://user-images.githubusercontent.com/9112811/96492655-377ae400-1201-11eb-9f0e-2be925c11be0.png)
In the case of a type `react.forward_ref`, the render has the name
![image](https://user-images.githubusercontent.com/9112811/96492746-55484900-1201-11eb-89fd-453ff2d603f8.png)